javascript-light-bulb/javascript-light-bulb/index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-<html class="on">
+<html>
   <head>
     <link rel="stylesheet" href="styles.css" />
     <title>Javascript Light Bulb</title>
   </head>
-  <body>
+  <body class='on'>
```

Comment:
should be double quotes for the attribute value

Reply:
Updated